package.json

```diff
@@ -29,6 +29,7 @@
     "gatsby-plugin-mdx": "^5.7.0",
     "gatsby-plugin-netlify": "^5.1.0",
     "gatsby-plugin-react-svg": "^3.3.0",
+    "gatsby-plugin-robots-txt": "^1.8.0",
```

Comment:
How come you add a plugin to `package.json`, but there's no change in `package-lock`? 🤔

Reply:
Whoops, returned :)